.linkcheckerrc

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[filtering]
+ignore=
+  http://localhost:8123/documentation/[0-9]+.[0-9]+.[0-9]+
```

Comment:
Why this ignore line? We don't want to check broken links on built documentation?

Reply:
It is specifically ignoring the references to previous release versions of the documentation which don't get built as part of this process but do sit on the web server.